### PR TITLE
New: OnApplicationUpdate Notifications

### LINKED ISSUE #2552: Pull Radarr commit 'New: OnApplicationUpdate Notifications (#6854)'
Pull https://github.com/Radarr/Radarr/commit/9e175e28efcfc6ac3e414649b955a10fb0e951e7

```
git cherry-pick -ex 9e175e28efcfc6ac3e414649b955a10fb0e951e7
```

This commit has 16 conflict(s)

You can ask me to `close all`, `open pr here` or `open pr everywhere`

[AB#2233](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/2233)

### PR DESCRIPTION
**Pull request by PearsonFlyer**
* Added translations

Fixes #2552
[common]

(cherry picked from commit 9e175e28efcfc6ac3e414649b955a10fb0e951e7)

#### Database Migration
YES - 054

#### Description
Adds "OnApplicationUpdate" notifications to all appropriate notification services.

#### Screenshot (if UI related)
image

#### Todos
- [ ] Tests
- [X] [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #2552

### CODE REVIEW

File: frontend/src/Settings/Notifications/Notifications/Notification.js [hunk 1]
```diff
@@ -139,9 +141,16 @@ class Notification extends Component {
             </Label>
         }
 
+        {
+          supportsOnApplicationUpdate && onApplicationUpdate &&
+            <Label kind={kinds.SUCCESS} >
+              On Application Update
```
Comment (bakerboy448) [inline frontend/src/Settings/Notifications/Notifications/Notification.js:147]: Translate (and knock out the rest while we're here)

ref https://github.com/Radarr/Radarr/blob/fda1ad237bc9d93e420f637ece0c09d524b8b44b/frontend/src/Settings/Notifications/Notifications/Notification.js#L131

File: src/NzbDrone.Core/Notifications/Notifiarr/Notifiarr.cs [hunk 2]
```diff
@@ -96,6 +96,18 @@ public override void OnHealthIssue(HealthCheck.HealthCheck healthCheck)
             _proxy.SendNotification(variables, Settings);
         }
 
+        public override void OnApplicationUpdate(ApplicationUpdateMessage updateMessage)
+        {
+            var variables = new StringDictionary();
+
+            variables.Add("Radarr_EventType", "ApplicationUpdate");
```
Comment (bakerboy448) [inline src/NzbDrone.Core/Notifications/Notifiarr/Notifiarr.cs:103]: Lidarr not Radarr

**Review by bakerboy448 — CHANGES_REQUESTED** [review]

File: frontend/src/Settings/Notifications/Notifications/Notification.js (hunk 1 above)
Comment (PearsonFlyer) [reply frontend/src/Settings/Notifications/Notifications/Notification.js]: I went ahead and did them all.

**Review by Qstick — CHANGES_REQUESTED** [review]
This looks pretty good, just a few small issues.

File: src/NzbDrone.Core/Notifications/ApplciationUpdateMessage.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+using System;
```
Comment (Qstick) [inline src/NzbDrone.Core/Notifications/ApplciationUpdateMessage.cs:1]: Name of this file should be corrected, spelling

**Review by Qstick — APPROVED** [review]

**Pull request merged**